frontend/src/global_styles/content/_enterprise.sass

```diff
@@ -13,19 +13,21 @@
   font-family: "Lucida Console", Monaco, monospace
   max-width: 560px
 
-.widget-box--blocks--upsell-buttons
-  display: flex
-  justify-content: flex-end
-
-  a.button,
-  a.button--link
-    text-decoration: none
-    margin: 0.625rem 0.625rem 0 0
-
-.widget-box--blocks--upsell-info-button
-  display: flex
-  justify-content: center
-  align-items: center
+.controller-homescreen .widget-boxes
+  .widget-box.upsell
+    padding: 0
+    background: var(--bgColor-accent-muted)
+    flex-basis: 100%
+    border: var(--borderWidth-thin) solid var(--borderColor-severe-muted)
+    border-radius: var(--borderRadius-medium)
+    background: var(--body-background)
+
+.widget-box--blocks--upsell-container
+  padding: 20px
```

Comment:
This value comes from the outer widget block. Not sure if we can adapt it to match the banner spec perfectly.

Reply:
I think it matches with the styles of other widgets on the page now, and looks good.